e2e/cypress/page-objects/fill-out-activity-page.js

```diff
@@ -73,30 +73,59 @@ class FillOutActivityPage {
     }
   };
 
-  fillStateStaff = (years, staffList, testDelete = false) => {
+  fillStateStaff = (
+    years,
+    staffList,
+    testDelete = false,
+    testBudget = false
+  ) => {
     cy.findByRole('heading', {
       name: /State Staff and Expenses/i,
       level: 3
     }).should('exist');
 
-    _.forEach(staffList, (staff, i) => {
+    for (let i = 0; i < staffList.length; i++) {
       staffExpensesPage.addStaff();
       staffExpensesPage.fillStaff({
         years,
         staffIndex: i,
-        title: staff.title,
-        description: staff.description,
-        costs: staff.costs,
-        ftes: staff.ftes
+        title: staffList[i].title,
+        description: staffList[i].description,
+        costs: staffList[i].costs,
+        ftes: staffList[i].ftes
       });
+      if (testBudget) {
+        break;
+      }
       staffExpensesPage.verifyStaff(
         i,
-        staff.title,
-        staff.description,
-        staff.costs,
-        staff.ftes
+        staffList[i].title,
+        staffList[i].description,
+        staffList[i].costs,
+        staffList[i].ftes
       );
-    });
+    }
+
+    // _.forEach(staffList, (staff, i) => {
```

Comment:
did the forEach version not work? can you remove this commented out code

Reply:
I couldn't use `break `to get out of a forEach() so I changed them to for loops

Reply:
ahhh could you make a comment saying that above the tests just so no one comes in and tries to make it a forEach again, and then remove the commented code.